build.sbt

```diff
@@ -54,28 +54,23 @@ lazy val warningSuppression = Seq(
 )
 
 lazy val publishSettings = Seq(
-  versionScheme := Some("pvp"),
+  versionScheme := Some("semver-spec"),
   publishMavenStyle := true,
   Test / publishArtifact := false,
   pomIncludeRepository := { x => false },
-  pomExtra := <url>http://chisel.eecs.berkeley.edu/</url>
+  pomExtra := <url>https://www.chisel-lang.org</url>
     <licenses>
       <license>
         <name>apache-v2</name>
         <url>https://opensource.org/licenses/Apache-2.0</url>
         <distribution>repo</distribution>
       </license>
-    </licenses>
-    <developers>
-      <developer>
-        <id>jackbackrack</id>
-        <name>Jonathan Bachrach</name>
-        <url>http://www.eecs.berkeley.edu/~jrb/</url>
-      </developer>
-    </developers>,
```

Comment:
Having _someone_ here may be good as there is then at least a hint at who to contact. This may be irrelevant given that GitHub exists. Clearly Jonathan is no longer the right person for this. However, you (Jack) may be.

We can revisit this in a future PR.